src/app/core/statistics/models/usage-report.model.ts

```diff
@@ -0,0 +1,45 @@
+import { autoserialize, inheritSerialization } from 'cerialize';
+import { typedObject } from '../../cache/builders/build-decorators';
+import { excludeFromEquals } from '../../utilities/equals.decorators';
+import { ResourceType } from '../../shared/resource-type';
+import { HALResource } from '../../shared/hal-resource.model';
+import { USAGE_REPORT } from './usage-report.resource-type';
+import { HALLink } from '../../shared/hal-link.model';
+import { deserialize, autoserializeAs } from 'cerialize';
+
+@typedObject
+@inheritSerialization(HALResource)
+export class UsageReport extends HALResource {
```

Comment:
Please add TypeDocs for this new class, and the `Point` interface in this same file

Reply:
Added typedocs